report_qweb_element_page_visibility/views/layouts.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <template id="minimal_layout" inherit_id="web.minimal_layout">
+        <xpath expr="//t[@t-if='subst is True']" position="replace">
+            <t t-if="subst is True">
+                <script>
+                    function subst() {
```

Comment:
Maybe you should to ttake the subst function code from 11.0 and add the visibility code to it https://github.com/OCA/OCB/blob/11.0/addons/web/views/report_templates.xml

Reply:
@jesusVMayor, I think to use replace position attribute it's never a good practice but I don't know another way to do in templates. Can you help me?
Thanks!

Reply:
@svazquezco maybe /pull/247 will give some hints, just from a user perspective view.